src/main/resources/xslt/cassandra/test-pom.xsl

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+    xmlns:pom="http://maven.apache.org/POM/4.0.0" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xalan="http://xml.apache.org/xslt" exclude-result-prefixes="pom xalan">
+
+    <xsl:output method="xml" indent="yes" xalan:indent-amount="2" />
+    <xsl:strip-space elements="*" />
+
+    <xsl:template match="@* | node()">
+        <xsl:copy>
+            <xsl:apply-templates select="@* | node()" />
+        </xsl:copy>
+    </xsl:template>
+
+    <!-- Force quarkus-cassandra-client-test.version property for any artifactId matching cassandra-quarkus-integration-tests -->
+    <xsl:template match="/pom:project/pom:dependencies/pom:dependency/pom:version">
+        <xsl:copy>
+            <xsl:if test="starts-with(../pom:artifactId/text(), 'cassandra-quarkus-integration-tests')"><xsl:apply-templates select="@*" />${quarkus-cassandra-client-test.version}</xsl:if>
+        </xsl:copy>
+    </xsl:template>
+    <xsl:template match="/pom:project/pom:profiles/pom:profile/pom:build/pom:plugins/pom:plugin[2]/pom:executions/pom:execution/pom:configuration/pom:appArtifact">
+        <xsl:copy><xsl:apply-templates select="@*" />com.datastax.oss.quarkus:<xsl:value-of select="/pom:project/pom:artifactId/text()"/>:${quarkus-cassandra-client-test.version}</xsl:copy>
+    </xsl:template>
```

Comment:
@jamesnetherton is this really necessary? Looks like it shouldn't be.

Reply:
Not 100% sure TBH. I just saw that the `<appArtifact>` thing referenced the plain `quarkus-cassandra-client.version` property and thought it should probably point at `quarkus-cassandra-client-test.version` instead.

Reply:
Thanks for checking, I'll have a look.

Reply:
@jamesnetherton that's fine, you don't need to use XSLT here. It's just it's trying to guess the property here and is picking up the Cassandra BOM one for it. Try changing the test version to something that doesn't match the BOM version and you'll see it using the proper one for the tests.

Reply:
Thanks for clarifying @aloubyansky. I have removed that part of the XSLT.